logger/scenario_logger_msgs/package.xml

```diff
@@ -1,20 +1,21 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>scenario_logger_msgs</name>
   <version>0.0.0</version>
   <description>The scenario_logger_msgs package</description>
   <maintainer email="shotahirama@todo.todo">shotahirama</maintainer>
 
   <license>TODO</license>
```

Comment:
Maybe adding the Apache License here? I also don't know if the we should be ensuring that the metadata above is up-to-date?
```suggestion
  <license>Apache License 2.0</license>
```